client/src/containers/VerifyEmailPage.tsx

```diff
@@ -59,7 +66,7 @@ const VerifyEmailPage = withI18n()((props: withI18nProps) => {
         <button
           className="button is-secondary"
           onClick={async () => {
-            setIsEmailResent(await AuthClient.resendVerifyEmail());
+            setIsEmailResent(await AuthClient.resendVerifyEmail(token));
           }}
         >
           <Trans>Resend verification email</Trans>
```

Comment:
```suggestion
          <Trans>Send new link</Trans>
```
Based on the [new designs](https://www.figma.com/file/Gv7RdLkCV21Z5FaY7RrRnH/Email-Alerts---Working-File-v2-%F0%9F%9B%A0%EF%B8%8F?type=design&node-id=39-1823&mode=design&t=iL55bjJ6gtVFQEaq-0) and conversations about the reset password flow, it sounds like Leslie wants all resend buttons to have this same text.

Reply:
oo good catch, thank u